document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -608,6 +641,35 @@ public void drop() {
     }
   }
 
+  private Map<String, String> getDocIdToTenantIdMap(BulkArrayValueUpdateRequest request) {
```

Comment:
This looks like a util method, which can go into an util class.

Reply:
No change, refactored. But, we can handle it.